blocklyc.html

```diff
@@ -67,25 +61,27 @@
         <script type="text/javascript" src="src/lib/bootstrap/core/js/bootstrap.min.js"></script>
         <script type="text/javascript" src="src/lib/bootstrap/plugins/bootbox.min.js"></script>
         <script type="text/javascript" src="src/ace/ace.js"></script>
+        <script type="text/javascript" src="src/lib/jquery.validate.min.js"></script>
 
         <!-- Javascript helpers -->
         <script type="text/javascript" src="src/utils.js"></script>
         <script type="text/javascript" src="src/propterm.js"></script>
         <script type="text/javascript" src="src/blocklypropclient.js"></script>
         <script type="text/javascript" src="src/blocklyc.js"></script>
         <script type="text/javascript" src="src/editor.js"></script>
+        <script type="text/javascript" src="src/localcompile.js"></script>
```

Comment:
Why is localcompile.js included here? Compilation is happening in a separate container with its own Rest interface.

Reply:
It doesn't need to be for this application.